--- alphashooter-0.0.3/Makefile~	2007-08-14 19:05:43.000000000 +0200
+++ alphashooter-0.0.3/Makefile	2007-08-14 19:05:41.000000000 +0200
@@ -21,18 +21,21 @@
 
 
 SHELL=/bin/sh
-prefix?=$(DESTDIR)/usr/local
+prefix?=$(DESTDIR)/usr
 exec_prefix=$(prefix)
-bindir=$(exec_prefix)/games
+bindir=$(exec_prefix)/bin
 DATADIR=$(prefix)/share/games
 INSTALL=install
 INSTALL_PROGRAM=$(INSTALL)
 INSTALL_DATA=$(INSTALL) -m 644
 
 CC=gcc
-LD=gcc
+LD=$(CC)
 NAME=alphashooter
 
+CFLAGS=
+LDFLAGS=
+
 SRCPATH=src
 DATAPATH=data
 DOCPATH=doc
@@ -58,8 +61,8 @@
 DEBDISTDIR=dists/unstable/main
 
 VERSION=0.0.3
-CFLAGS=-Wall -g -ggdb -O3 -D ALPSHOOT_VERSION="\"$(VERSION)\""
-LDFLAGS=-lm -D ALPSHOOT_VERSION="\"$(VERSION)\""
+CFLAGS+=-Wall -g -ggdb -O3 -D ALPSHOOT_VERSION="\"$(VERSION)\""
+LDFLAGS+=-lm -D ALPSHOOT_VERSION="\"$(VERSION)\""
 
 export INSTALL=install
 export INSTALL_PROGRAM=$(INSTALL)
